openbsd-games.db

```diff
@@ -4440,7 +4440,7 @@ Cover	Ruggnar_cover.png
 Engine	MonoGame
 Setup	fnaify
 Runtime	fnaify
-Store	https://sirill.itch.io/ruggnar
+Store	https://sirill.itch.io/ruggnar https://store.steampowered.com/app/1303050 https://www.gog.com/game/ruggnar
```

Comment:
this one might need a note that recent versions are built with dotnet and don't run anymore... would not recommend buying this one, as it might never run on OpenBSD anymore